buildlib/pr/io_demo/io-demo.yml

```diff
@@ -122,59 +39,78 @@ jobs:
           pathToPublish: '$(Build.ArtifactStagingDirectory)'
           artifactName: drop_$(Build.BuildId)
 
-  - job: test
-    dependsOn: io_build
-    workspace:
-      clean: all
-
-    pool:
-      name: MLNX
-      demands: ${{ parameters.demands }}
-
-    strategy:
-      matrix:
-        ${{ each test in parameters.tests }}:
-          ${{ test.Key }}:
-            test_name: ${{ test.Value.test_name }}
-            test_args: ${{ test.Value.args }}
-            test_time: ${{ test.Value.duration }}
-            test_intefrafe: ${{ test.Value.interface }}
-            test_ucx_tls: ${{ test.Value.tls }}
-            test_extra_run_args: ${{ test.Value.extra_run_args }}
-            initial_delay: ${{ coalesce(test.Value.initial_delay, parameters.initial_delay) }}
-            test_interference: ${{ test.Value.interference }}
-      maxParallel: 1
+  - template: jobs-io-demo.yaml
+    parameters:
+      demands: "ucx_iodemo -equals yes"
+      name: "with_interference"
+      tests:
+        "tag match on CX4":
+          duration: 600
+          interface: $(roce_iface_cx4)
+          tls: "rc_x"
+          test_name: tag_cx4_rc
+          extra_run_args: ""
+        "tag match on CX6/RC":
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "rc_x"
+          test_name: tag_cx6_rc
+          extra_run_args: ""
+        "active messages on CX6/RC":
+          args: "-A"
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "rc_x"
+          test_name: am_cx6_rc
+          extra_run_args: ""
+        "tag match on CX6/DC":
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "dc_x"
+          test_name: tag_cx6_dc
+          extra_run_args: ""
+        "active messages on CX6/DC with data validation":
+          args: "-A -q"
+          duration: 600
+          interface: $(roce_iface_cx6)
+          tls: "dc_x"
+          test_name: am_validate_cx6_dc
+          extra_run_args: ""
 
-    variables:
-      workspace: drop_$(Build.BuildId)
-      io_demo_exe: drop_$(Build.BuildId)/install/bin/io_demo
-      cycles: ${{ parameters.cycles }}
-      downtime: ${{ parameters.downtime }}
-      uptime: ${{ parameters.uptime }}
+  - template: jobs-io-demo.yaml
+    parameters:
+      demands: "ucx_iodemo_without_interference -equals yes"
```

Comment:
maybe call this tag "ucx_iodemo_shared"

Reply:
too, shared - no additional information , but ucx_iodemo_without_interference say that there is label for iodemo without interference

Reply:
ok